### DISCUSSION TITLE
How to compile Uno libraries with VS 2019 for Mac

### DISCUSSION THREAD

**Discussion by ericleroy01** [unoplatform/uno#6014] [Q&A] [answered]
Hi,

I would like to fix issue https://github.com/unoplatform/uno/issues/5763
But currently I'm able to compile Uno librairies.

Is there an available procedure ?

**Comment by jeromelaban** [comment, marked as answer]
You can follow the guide here: https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/building-uno-macos.md
